src/components/ForumCard.js

```diff
@@ -0,0 +1,262 @@
+import React, { useRef, useState, useEffect } from "react"
+import { useAuth } from "../contexts/AuthContext"
+import { Link, useHistory } from "react-router-dom"
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import NavigationBar from './NavigationBar';
+import Box from "@material-ui/core/Box";
+import { useParams } from 'react-router-dom';
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const Forums = () => {
```

Comment:
this file is called `ForumCard` but the component is called `Forums`

Reply:
also, `ForumCard` isn't the best fit for what this page is, something like `Forum` is better, open to suggestions